src/main/scala/org/allenai/plugins/VersionInjectorPlugin.scala

```diff
@@ -40,6 +40,14 @@ object VersionInjectorPlugin extends AutoPlugin {
       "gitDescribe",
       "The description of the current git commit"
     )
+    val gitMostRecentCommit = TaskKey[String](
```

Comment:
I would actually change how the `gitSha1` task worked, and just have it use the local project settings. Since we only use `Version` for informative purposes, this would be a net win overall.